src/app/public/modules/ag-grid/cell-editors/cell-editor-number/cell-editor-number.component.spec.ts

```diff
@@ -0,0 +1,125 @@
+import {
+  async,
+  ComponentFixture,
+  TestBed
+} from '@angular/core/testing';
+
+import {
+  SkyAppTestModule
+} from '@skyux-sdk/builder/runtime/testing/browser';
+
+import {
+  expect
+} from '@skyux-sdk/testing';
+
+import {
+  ICellEditorParams
+} from 'ag-grid-community';
+
+import {
+  SkyAgGridCellEditorNumberComponent
+} from '../cell-editor-number';
+
+describe('SkyCellEditorNumberComponent', () => {
+  let fixture: ComponentFixture<SkyAgGridCellEditorNumberComponent>;
+  let component: SkyAgGridCellEditorNumberComponent;
+  let nativeElement: HTMLElement;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        SkyAppTestModule
```

Comment:
The `SkyAppTestModule` is ideal for SPA unit testing, but adds a lot of bulk to library testing.

Instead of using the `SkyAppTestModule`, most of our components implement a "fixtures" module. See: https://github.com/blackbaud/skyux-forms/blob/master/src/app/public/modules/toggle-switch/toggle-switch.component.spec.ts#L46

(I noticed the `SkyAppTestModule` in a few other specs.)